kubernetes/tests/kubernetes/kubernetes.tests.bom

```diff
@@ -116,17 +116,7 @@ brooklyn.catalog:
           command: |
             kubectl get deployments | grep workload- | wc -l
           assertOut:
-            contains: '0'
-        - type: org.apache.brooklyn.test.framework.TestHttpCall
-          name: "14. Check Prometheus UI is Reachable"
-          url:
-            $brooklyn:formatString:
-              - "http://%s:%s"
-              - $brooklyn:entity("prometheus").attributeWhenReady("host.address")
-              - $brooklyn:entity("prometheus").attributeWhenReady("kubernetes.service.port")
-          applyAssertionTo: status
-          assert:
-          - equals: 200
+            contains: '0'        
```

Comment:
I don't think we want to run these tests against a Kubernetes cluster that cannot be SSHed to.

I think it is more likely we run them against clouds on which we don't open the high value ports, so I agree with Marks suggestion.